### ISSUE TITLE
Error executing QTiles 1.4.3 in QGIS 2.6.0 Windows

### ISSUE THREAD

**Issue by ozjimbob** [nextgis/QTiles#23]
Upon attempting to render a layer to tiles in QTiles 1.4.3 in QGIS 2.6.0 Windows 32-bit, it immediately fails with the following error:

```
Traceback (most recent call last):
  File "C:/Users/grantw1/.qgis2/python/plugins\qtiles\qtilesdialog.py", line 209, in accept
    writeViewer)
  File "C:/Users/grantw1/.qgis2/python/plugins\qtiles\tilingthread.py", line 108, in __init__
    self.settings.setBackgroundColor(self.color)
TypeError: QgsMapSettings.setBackgroundColor(QColor): argument 1 has unexpected type 'long'


Python version:
2.7.4 (default, Apr  6 2013, 19:54:46) [MSC v.1500 32 bit (Intel)]


QGIS version:
2.6.0-Brighton Brighton, exported

Python path: ['C:/PROGRA~2/QGISBR~1/apps/qgis/./python/plugins\\processing', 'C:/PROGRA~2/QGISBR~1/apps/qgis/./python', u'C:/Users/grantw1/.qgis2/python', u'C:/Users/grantw1/.qgis2/python/plugins', 'C:/PROGRA~2/QGISBR~1/apps/qgis/./python/plugins', 'C:\\PROGRA~2\\QGISBR~1\\bin\\python27.zip', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\DLLs', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\plat-win', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\lib-tk', 'C:\\PROGRA~2\\QGISBR~1\\bin', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\PIL', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\jinja2-2.7.2-py2.7.egg', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\markupsafe-0.23-py2.7-win32.egg', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\python_dateutil-2.2-py2.7.egg', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\pytz-2014.2-py2.7.egg', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\win32', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\win32\\lib', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\Pythonwin', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\Shapely-1.2.18-py2.7-win32.egg', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\six-1.6.1-py2.7.egg', 'C:\\PROGRA~2\\QGISBR~1\\apps\\Python27\\lib\\site-packages\\wx-2.8-msw-unicode', 'C:\\PROGRA~2\\QGISBR~1\\apps\\qgis\\python\\plugins\\fTools\\tools']
```

This occurs for me for both polygon and point layers, whether I render to JPG or PNG, and no matter what PNG background transparency I select.

**Comment by anitagraser** [comment]
I get the same error in 2.6.1

**Comment by simgislab** [comment]
still valid?

**Comment by gstaab** [comment]
This seems to have been fixed with 1.4.4 of QTiles. Sorry, should have left a comment.

**Comment by simgislab** [comment]
cool, please reopen if necessary
